WalletWasabi.Fluent/Rpc/WasabiJsonRpcService.cs

```diff
@@ -48,6 +51,19 @@ public object[] GetUnspentCoinList()
 			}).ToArray();
 		}
 
+		[JsonRpcMethod("listpockets")]
+		public object[] GetPockets()
+		{
+			var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+			SourceList<PocketViewModel> pocketS = new();
```

Comment:
Nit: rename the variable to `pockets`

Reply:
Do not use viewmodels in the RPC code, one interface cannot depend on another interface.